tests/lib/prepare-restore.sh

```diff
@@ -319,6 +319,8 @@ prepare_project_each() {
             systemctl start systemd-journald.service
             ;;
         *)
+            # rotate and sync 'override' each other if used in a single command
```

Comment:
I would tweak the comment a bit, i.e. clarify that it is not possible to use `journalctl --sync --rotate` because thats a systemd internal issue not something conceptually.

Reply:
Right, good point.